rust/protocol/src/identity_key.rs

```diff
@@ -120,6 +150,7 @@ impl IdentityKeyPair {
         result.into_boxed_slice()
     }
 
+    /// Generate a signature for a provided identity key `other` (*not* `self`)!
```

Comment:
Similarly, perhaps "Generate a signature claiming that `other` represents the same user as `self`."

Reply:
Done! I also clarified that `signature` in `IdentityKey::verify_alternate_identity` must be calculated from `IdentityKeyPair::sign_alternate_identity`.